src/index.ts

```diff
@@ -411,7 +411,7 @@ function dontParse(paths: webpack.Module["noParse"]): webpack.Configuration {
 // to `staticPaths`.
 function webpackPluginServe({
   staticPaths,
-  historyApiFallback: historyFallback,
+  historyApiFallback,
```

Comment:
This will be `undefined` by default and that was the issue as webpack-plugin-serve has a stricter type validation now.